admin/includes/functions/html_output.php

```diff
@@ -175,10 +175,14 @@ function zen_black_line() {
 ////
 // Output a separator either through whitespace, or with an image
   function zen_draw_separator($image = 'pixel_black.gif', $width = '100%', $height = '1') {
-    if (substr(rtrim($width), -1) != "%") $width = $width . 'px';
-    return zen_image(DIR_WS_IMAGES . $image, '', '', $height, 'style="width:' . $width . ';"');
+	if (!empty($width)) {
+		if (substr(rtrim($width), -1) != "%") $width = $width . 'px';
```

Comment:
Line 179 should be changed to
```php
        if (substr(rtrim($width), -1) !== '%') {
            $width = $width . 'px;'
        }
```
1. Using exactly-not-equal-to
2. Using single-quotes around that character string.
3. Adhering to the PSR that indicates that no one-liners are allowed on conditional statements.

Reply:
Changes made as requested.